drivers/dai/nxp/sai/sai.h

```diff
@@ -38,7 +38,8 @@ LOG_MODULE_REGISTER(nxp_dai_sai);
 
 /* used to retrieve a clock's ID using its index generated via _SAI_CLOCK_INDEX_ARRAY */
 #define _SAI_GET_CLOCK_ID(clock_idx, inst)\
-	DT_INST_CLOCKS_CELL_BY_IDX(inst, clock_idx, name)
+	COND_CODE_1(DT_INST_PHA_HAS_CELL_AT_IDX(inst, clocks, clock_idx, name),\
+		    (DT_INST_CLOCKS_CELL_BY_IDX(inst, clock_idx, name)), (0x0))
```

Comment:
run `clang-format -i drivers/dai/nxp/sai/sai.h` and will format the file as expected by CI :)
Otherwise, LGTM

Reply:
@iuliana-prodan clang-format wants this change:

```
 /* used to retrieve a clock's ID using its index generated via _SAI_CLOCK_INDEX_ARRAY */
-#define _SAI_GET_CLOCK_ID(clock_idx, inst)\
+#define _SAI_GET_CLOCK_ID(clock_idx, inst)                                                         \
 »      COND_CODE_1(DT_INST_PHA_HAS_CELL_AT_IDX(inst, clocks, clock_idx, name),\
 »      »           (DT_INST_CLOCKS_CELL_BY_IDX(inst, clock_idx, name)), (0x0))

```

I prefer to keep the existing style of the header file. If we want we can sent a separate patch later as they are more than 10 other instances where this should be fixed.